examples/simple_scene.py

```diff
@@ -10,21 +10,27 @@
 
 scene = vv.Scene()
 
-t1 = vv.Mesh(vv.Geometry(), vv.TriangleMaterial())
+
+class Triangle(vv.WorldObject):
+    def __init__(self, material):
+        super().__init__()
+        self.material = material
```

Comment:
I like that you moved it here. :) it's a weird little object, with hardcoded geometry